desci-models/src/ResearchObject.ts

```diff
@@ -120,59 +120,60 @@ export interface ResearchObjectV1Attributes {
 }
 
 export enum ResearchObjectComponentType {
-  DATA_BUCKET = 'data-bucket',
-  UNKNOWN = 'unknown',
-  PDF = 'pdf',
-  CODE = 'code',
-  VIDEO = 'video',
-  TERMINAL = 'terminal', // not used, TODO: remove
-  DATA = 'data',
-  LINK = 'link', // external link
+  DATA_BUCKET = "data-bucket",
+  UNKNOWN = "unknown",
+  PDF = "pdf",
+  CODE = "code",
+  VIDEO = "video",
+  TERMINAL = "terminal", // not used, TODO: remove
+  DATA = "data",
+  LINK = "link", // external link
 }
 
 export enum ResearchObjectComponentDocumentSubtype {
-  RESEARCH_ARTICLE = 'research-article',
-  PREREGISTERED_REPORT = 'preregistered-report',
-  PREREGISTERED_ANALYSIS_PLAN = 'preregistered-analysis-plan',
-  SUPPLEMENTARY_INFORMATION = 'supplementary-information',
-  PRESENTATION_DECK = 'presentation-deck',
-  OTHER = 'other',
-}
-
-export enum ResearchObjectComponentDocumentSubtypeV2 {
-  AUTHOR_ACCEPTED = 'author-accepted',
-  PREPRINT = 'preprint',
-  REVIEW_REPORT = 'review-report',
+  RESEARCH_ARTICLE = "research-article",
+  PREREGISTERED_REPORT = "preregistered-report",
+  PREREGISTERED_ANALYSIS_PLAN = "preregistered-analysis-plan",
+  SUPPLEMENTARY_INFORMATION = "supplementary-information",
+  PRESENTATION_DECK = "presentation-deck",
+  AUTHOR_ACCEPTED = "author-accepted",
+  PREPRINT = "preprint",
+  REVIEW_REPORT = "review-report",
+  MANUSCRIPT = "manuscript",
+  OTHER = "other",
 }
 
 export enum ResearchObjectComponentDataSubtype {
-  PROCESSED_DATA = 'processed-data',
-  RAW_DATA = 'raw-data',
-  IMAGE = 'image',
-  OTHER = 'other',
+  PROCESSED_DATA = "processed-data",
+  RAW_DATA = "raw-data",
+  IMAGE = "image",
+  OTHER = "other",
 }
 
 export enum ResearchObjectComponentCodeSubtype {
-  CODE_SCRIPTS = 'code-scripts',
-  SOFTWARE_PACKAGE = 'software-package',
-  OTHER = 'other',
+  CODE_SCRIPTS = "code-scripts",
+  SOFTWARE_PACKAGE = "software-package",
+  OTHER = "other",
 }
 
 export enum ResearchObjectComponentLinkSubtype {
-  COMMUNITY_DISCUSSION = 'community-discussion',
-  VIDEO_RESOURCE = 'video-resource',
-  EXTERNAL_API = 'external-api',
-  RESTRICTED_DATA = 'restricted',
-  PRESENTATION_DECK = 'presentation-deck',
-  OTHER = 'other',
+  COMMUNITY_DISCUSSION = "community-discussion",
+  VIDEO_RESOURCE = "video-resource",
+  EXTERNAL_API = "external-api",
+  RESTRICTED_DATA = "restricted",
+  PRESENTATION_DECK = "presentation-deck",
+  OTHER = "other",
 }
 
 export type ResearchObjectComponentSubtypes =
   | ResearchObjectComponentDocumentSubtype
-  | ResearchObjectComponentDocumentSubtypeV2
   | ResearchObjectComponentDataSubtype
   | ResearchObjectComponentCodeSubtype
-  | ResearchObjectComponentLinkSubtype;
+  | ResearchObjectComponentLinkSubtype
+  | ResearchObjectComponentType.PDF
```

Comment:
Why are types being added as subtypes @Electromorphous maybe I missed this part in the standup -- why is this necessary? I see it in multiple places in this PR